public_html/utils/UserManager.php

```diff
@@ -49,12 +48,6 @@ static function addUser($firstname, $lastname, $email, $password) {
         $user->username = $email;
         return $db->insertNewUser($user, $password); //return true or false if insert succeeded
     }
-	
-	static function updateSettings($username, $user_settings) {
```

Comment:
This is needed by the settings page. Undo this function delete

Reply:
Added back